sdk/02_tutorial/07_verification.md

```diff
@@ -23,28 +23,28 @@ The Verifier also needs an account and DID, but the tutorial omits these steps.
 
 1. Take a `VerifiablePresentation` object supplied by a <span className="label-role holder">Holder</span>
 2. Verify that its data is correct
-3. Verify that the presentation is valid, i.e., its hash exists on-chain and the presentation hasn't been revoked
-4. Verify that the <span className="label-role holder">Holder</span> sending the `Credential` owns it
+3. Verify that the <span className="label-role holder">Holder</span> of the `Credential` has authorised and consented to its use in the current context by checking the presentation's signature and attributes.  
+4. Verify the authenticity and validity of the credential by checking its on-chain proof created by the Issuer and ensuring it hasn't been revoked since
```

Comment:
```suggestion
4. Verify the authenticity and validity of the credential by checking its on-chain proof created by the Issuer and ensuring it hasn't been revoked since.
```